pyiron_atomistics/vasp/base.py

```diff
@@ -835,27 +773,12 @@ def get_final_structure_from_file(self, cwd, filename="CONTCAR"):
         Returns:
             pyiron.atomistics.structure.atoms.Atoms: The final structure
         """
-        filename = posixpath.join(cwd, filename)
-        if self.structure is None:
-            try:
-                output_structure = read_atoms(filename=filename)
-                input_structure = output_structure.copy()
-            except (IndexError, ValueError, IOError):
-                raise IOError("Unable to read output structure")
-        else:
-            input_structure = self.structure.copy()
-            try:
-                output_structure = read_atoms(
-                    filename=filename,
-                    species_list=input_structure.get_parent_symbols(),
-                )
-                input_structure.cell = output_structure.cell.copy()
-                input_structure.positions[self.sorted_indices] = (
-                    output_structure.positions
-                )
-            except (IndexError, ValueError, IOError):
-                raise IOError("Unable to read output structure")
-        return input_structure
+        return get_final_structure_from_file(
+            working_directory=cwd,
+            filename=filename,
+            structure=self.structure,
+            sorted_indices=self.sorted_indices,
+        )
```

Comment:
Just import from the output.py? why is this necessary in base.py?

Reply:
Primarily for backwards compatibility, I am not sure if anybody is using this function at the moment.

Reply:
I think if someone cared enough, they'd use ASE or pymatgen to read from file and convert it over, not worth the additional obfuscation/confusion potential this has. Anyone that knows how to dig this deep for a structure reader probably doesn't need it. my2c